Pipfile

```diff
@@ -13,6 +13,7 @@ pytest-cov = "*"
 pylint = "*"
 coverage = "*"
 pyinstaller = "*"
+typer = {extras = ["all"], version = "*"}
```

Comment:
Perfect, love it

Feedback: Move this to normal packages, this is required by users and is not a dev tool unlike the rest of our developer dependencies so this can fall under packages